healthcheck/core.go

```diff
@@ -581,14 +581,21 @@ func (s *Server) notifier() {
 // error and giving up after MaxFailures.
 func (s *Server) send() error {
 	failures := 0
-	for err := s.sendBatch(s.batch); err != nil; {
```

Comment:
Yeah, that's a nice bug... guess there's missing test coverage :)

Reply:
@4a6f656c Done it your way. :) Thanks.

Adjusted the order of failure++ and log though, to ensure "Send failed 0 times" never happen.